en/flying/missions.md

```diff
@@ -18,6 +18,28 @@ For more information see the [QGroundControl User Guide](https://docs.qgroundcon
 
 ![planning-mission](../../assets/flying/planning_mission.jpg)
 
+### Mission Feasibility Checks

```

Comment:
I'm not sure if this section is better here or in http://docs.px4.io/main/en/flight_modes/mission.html. 

Generally we use this mission planning page for planning - it has always been a bit "thin" because we assume that QGC tells you most of what you need to know - i.e. what commands you can use. It is only recently I have expanded it a bit. We use the flight mode page (http://docs.px4.io/main/en/flight_modes/mission.html) for more detailed information that you might use both for manually planning missions and to understand the mechanics / debugging / companion missions etc.

I think that where you have put it is the right place since you probably should know this if you're planning a mission. But then probably you also need it cross linked from the mode page?

Note, I need to put some hard thought into this whole page. Never been a priority because most of the time you can pick up what you need to know in QGC quite easily.

Reply:
>My leaning is to put this in http://docs.px4.io/main/en/flight_modes/mission.html and have a small section in this page.

Ah yes, that makes sense. The checks are clearly a "PX4 thing" and thus should live in the flight mode section. And here only reference that one, and mention that there are checks being run at mission upload/vehicle boot that need to pass to fly the mission. But really keep the flying/mission page to the bare minimum. 

The link from the flying/mission page to flight_modes/mission is missing atm.

>what are we hoping a user to know that they have to do when mission planning with this information?

I think it is important for users to know that there is some mission checking going on, and give them roughly an idea what the checks involve. Plus what to change if they have failures.

Reply:
@sfuhrer I reduced the section to be "high level" and linked.